etcdctl/ctlv3/command/migrate_command.go

```diff
@@ -220,7 +220,7 @@ func writeKeys(w io.Writer, n *store.NodeExtern) uint64 {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprintln(w, string(b))
```

Comment:
I read through https://golang.org/src/fmt/print.go. If seems the only difference is that println adds a `\n`. do you have an example?